modules/standard/Random.chpl

```diff
@@ -667,11 +843,21 @@ module Random {
         //if arr.domain.type.strideable then
         //  compilerError("Shuffle requires non-strideable 1-D array");
 
+        if parSafe then
```

Comment:
shuffle() feels like another slightly weird method for a random stream...  seems like it would be more natural as either a standalone routine or as a method on an array that took a randomstream of integers as its argument.